db/seeds.rb

```diff
@@ -80,7 +85,18 @@ def create_loan_requests_for_each_borrower(quantity)
                                                 repayment_rate: "weekly",
                                                 repayment_begin_date: repayment_begin_date)
         puts "created loan request #{request.title} for #{borrower.name}"
-        puts "There are now #{LoanRequest.all.count} requests"
+        puts "There are now #{LoanRequest.count} requests"
+      end
+    end
+  end
+
+  def create_orders(quantity)
+    quantity.times do
+      borrowers.each do |borrower|
+        lender = User.where(role: 0).order("RANDOM()").take(1).first
+        order = Order.create(cart_items: { "#{borrower.id}" => "25" }, user_id: lender.id)
+        order.update_contributed(lender)
+        puts "Created Order for Borrower #{borrower.name} by Lender #{lender.name}"
```

Comment:
Line is too long. [83/80]<br>Do not write to stdout. Use Rails' logger if you want to log.